src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/TagHelperDescriptorCache.cs

```diff
@@ -17,5 +20,117 @@ internal static class TagHelperDescriptorCache
 
         internal static void Set(int hashCode, TagHelperDescriptor descriptor) =>
             s_cachedTagHelperDescriptors.Set(hashCode, descriptor);
+
+#nullable enable
```

Comment:
Feel free to just remove the `disable` at the top of the file, we don't mind :)

Reply:
I didn't want to audit the implementation and all references for NRT-ness :).